test.html

```
@@ -0,0 +1,16 @@
+  <body>
+    <h1>Welcome
+    <img src="image.jpg">
+    <p>This paragraph is missing its closing tag
+    <a href="about.html">About us
+    <ul>
+      <li>Item 1
+      <li>Item 2
+    </ul
+  </body>
+</html>
+<html><body><h1>Oops<img src="a.png"></body></html>
```

HTMLHint (1.5.0)

[error] Tag must be paired, missing: [ </h1> ], start tag match failed [ <h1> ] on line 16.

(tag-pair)